src/items/functions.md

```diff
@@ -174,12 +174,15 @@ extern "C" fn new_i32() -> i32 { 0 }
 let fptr: extern "C" fn() -> i32 = new_i32;
 ```
 
-Functions with an ABI that differs from `"Rust"` do not support unwinding in the
-exact same way that Rust does. Therefore, unwinding past the end of functions
-with such ABIs causes the process to abort.
+Functions with an ABI that differs from `"Rust"` do not support unwinding in
+the exact same way that Rust does. Therefore, unwinding past the end of
+functions with such ABIs results in undefined behavior, and will differ between
+platforms and compiler editions. If you cannot guarantee this will not happen,
+you should use [`catch_unwind`] or similar to abort the program instead if
+unwinding would pass through the function.
 
-> **Note**: The LLVM backend of the `rustc` implementation
-aborts the process by executing an illegal instruction.
+> **Note**: Currently, in nightly `rustc` the LLVM backend aborts the process
+> by executing an illegal instruction. However, this is not the behavior in stable.
```

Comment:
We generally don't include notes about specific versions of the rustc compiler in the reference.  I'm also not sure what this is referring to, since it no longer aborts on unwind.